web-ethereum/dapp2/build.sh

```diff
@@ -10,6 +10,6 @@ $FLUTTER build web --web-renderer canvaskit
 
 # move js to hash named file
 uuid=$(uuidgen)
-sed -i '' "s/main.dart.js/${uuid}.main.dart.js/g" build/web/index.html
```

Comment:
What do the empty quotes do here?

Reply:
I'm not sure, but it wouldn't work with the default linux sed.   I switched it out for double quotes which does work, and allows for the variable replacement.